packages/@lwc/style-compiler/src/__tests__/index.spec.ts

```diff
@@ -60,26 +48,15 @@ function testFixture(fixtureDir: string) {
             const { code } = transform(actualSource, actualPath, config);
             result = code;
         } catch (err) {
-            if (expectedSource) {
-                throw err;
-            } else {
-                error = normalizeError(err);
-            }
+            error = normalizeError(err);
         }
 
-        if (expectedSource) {
-            expect(result).toBe(expectedSource);
-
+        if (error) {
```

Comment:
Let remove this change from this PR as it is discussed in https://github.com/salesforce/lwc/pull/2289.